src/python/turicreate/toolkits/_pre_trained_models.py

```diff
@@ -199,30 +199,40 @@ class StyleTransferTransformer():
 
     def __init__(self):
         self.name = 'resnet-16'
-        self.source_url = _urlparse.urljoin(MODELS_URL_ROOT, 'resnet-16.params')
-        self.source_md5 = 'ac232afa6d0ead93a8c75b6c455f6dd3'
-        self.model_path = _download_and_checksum_files([
-            (self.source_url, self.source_md5)
-        ], _get_cache_dir())[0]
+        self.source_md5 = { 'mxnet' : 'ac232afa6d0ead93a8c75b6c455f6dd3',
+                            'coreml': 'e0f3adaa9952ecc7d96f5e4eefb0d690' }
 
 
-    def get_model_path(self):
-        return self.model_path
+    def get_model_path(self, format):
+        assert(format in ('coreml', 'mxnet'))
+        if (format in 'coreml') :
+            url =  _urlparse.urljoin(MODELS_URL_ROOT, 'transformer.mlmodel')
```

Comment:
Should I worry that the base names are different between CoreML and MXNet? Is this the only transformer network we'll ever publish here?